ansible/idr-elasticsearch.yml

```diff
@@ -70,6 +81,20 @@
       group: root
       mode: 0755
 
+
+
+  - name: Add elastic nodes to instances_nodes
+    set_fact:
+      instances_nodes: "{{instances_nodes + [( {'name' : 'searchengine_elasticsearch_node'+item, 'dns': ['searchengine_elasticsearch_node'+item,'localhost'],'ip': '127.0.0.1'})] }}"
```

Comment:
The playbook failed to execute against `test120-searchengine` with the following error

```
The task includes an option with an undefined variable. The error was: 'instances_nodes' is undefined

The error appears to be in '/tmp/management_tools/idr/deployment/ansible/idr-elasticsearch.yml': line 86, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - name: Add elastic nodes to instances_nodes
     ^ here"
```

Reply:
@sbesson, I have pushed a fix for this issue, could you please try again?

Reply:
Same error with `instances` in the next task.